src/test/java/com/exactpro/epfast/ByteBuffProcessor.java

```diff
@@ -0,0 +1,9 @@
+package com.exactpro.epfast;
+
+import com.exactpro.epfast.decoder.OverflowException;
+import io.netty.buffer.ByteBuf;
+
+@FunctionalInterface
+public interface ByteBuffProcessor {
```

Comment:
rename to ByteBuff to ByteBuf in all names. I've misspelled it previously, sorry.